doc/source/tokudb/tokudb_variables.rst

```diff
@@ -911,6 +921,30 @@ output:
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: Off
+
+Enables the fast updates feature for the UPDATE statement. Fast update involves
+queries optimization to avoid random reads during their execution.
+
+.. variable:: tokudb_enable_fast_upsert
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: Off
+
+Enables the fast updates feature for the INSERT statement. Fast update involves
+queries optimization to avoid random reads during their execution.
```

Comment:
By this description it sounds that it should be just always enabled unconditionally, which is not the case

@georgelorchpercona